### PR TITLE
chores: add new adopters

### PR DESCRIPTION
**Pull request by milasuperstar**
Signed-off-by: Mila Wu <wumila1996@gmail.com>

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:
* Need to update Chaos Dashboard component, related issue:
* Need to cheery-pick to the release branch

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -6,30 +6,35 @@ If you are a Chaos Mesh adopter, and are willing to share your Chaos Mesh story,
...
   - To ensure the reliability of cloud storage products, they use Chaos Mesh to perform chaos tests on metadata and the underlying storage system under conditions such as: single point of failure of services, network abnormality, abnormal resource consumption (CPU, memory, I/O), etc.
 - [S.J. Distributors](https://www.sjfood.com/)
 - [Tencent](https://www.tencent.com/en-us)
-  - After Tencent Interactive Entertainment migrated their online operations to the Tencent Cloud Kubernetes engine, they wished to provide users with a more stable and reliable experience, which is why they introduced Chaos Mesh. Tencent mainly use Chaos Mesh to simulate the following types of failures:
+  - After Tencent Interactive Entertainment migrated their online operations to the Tencent Cloud Kubernetes engine, they wished to provide users with a more stable and reliable experience, which is [why they introduced Chaos Mesh](https://chaos-mesh.org/blog/Securing-Online-Gaming-Combine-Chaos-Engineering-with-DevOps-Practices/).
+  - Tencent mainly use Chaos Mesh to simulate the following types of failures:
```
Comment (fewdan) [inline ADOPTERS.md:37]: it should be "Tencent uses"?

File: ADOPTERS.md [hunk 2]
```diff
@@ -6,30 +6,35 @@ If you are a Chaos Mesh adopter, and are willing to share your Chaos Mesh story,
...
 - [ByteDance](https://bytedance.com/en/)
   - ByteDance is a technology company operating a range of content platforms including TikTok, Lark, Helo, Vigo Video, Douyin, and Huoshan in over 150 markets and 75 languages.
   - ByteDance's self-developed chaos engineering platform is mainly used by the company's own technology system. As there are some cloud-native deployment services involved, they integrated Chaos Mesh as the underlying fault injection engine, which is a key supplement to ByteDance’s chaos engineering platform.
 - [Celo](https://celo.org/)
 - [Dailymotion](https://www.dailymotion.com/)
-- [DataStax Fallout](https://github.com/datastax/fallout)
+- [DataStax](https://www.datastax.com/) 
+  - DataStax utilized Chaos Mesh to create a tool for running local or large scale remote based distributed correctness, verification and performance tests: [Fallout](https://github.com/datastax/fallout), which they use to [test AstraDB](https://youtu.be/Kw7gMurHJnQ), a multi-cloud DBaaS built on Apache Cassandra
```
Comment (fewdan) [inline ADOPTERS.md:16]: Is "utilized" inappropriate? Should we use the past tense here?

File: ADOPTERS.md (hunk 1 above)
Comment (milasuperstar) [reply ADOPTERS.md:37]: Ah, good catch, will update the doc :)

File: ADOPTERS.md (hunk 2 above)
Comment (milasuperstar) [reply ADOPTERS.md:16]: Well, they **used** Chaos Mesh to create Fallout the tool, and **uses** this tool to test AstraDB, so I reckon we should use past tense here? 🤔

**Review by g1eny0ung — APPROVED** [review]

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**